app/javascript/gobierto_plans/webapp/pages/GroupsByTerm.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <ul class="planification-table">
+    <li class="planification-table__li">
+      <div class="planification-table__li--content">
+        <div
+          v-for="[id, { name }] in columns"
+          :key="id"
+          class="planification-table__th"
+          @click="handleTableHeaderClick(id)"
+        >
+          <template v-if="id === 'length'">
+            <NumberLabel :level="lastLevel" />
+          </template>
+          <template v-else>
+            {{ name }}
+          </template>
+
+          <SortIcon
+            v-if="currentSortColumn === id"
+            :direction="getSorting(id)"
+          />
+        </div>
+      </div>
+    </li>
+    <template v-for="{ key, level } in termsSorted">
+      <GroupsByTermTableRowRecursive
+        v-if="level === 0"
+        :id="key"
+        :key="key"
+        :groups="groups"
+      />
+    </template>
+  </ul>
+</template>
+
+<script>
+import NumberLabel from "../components/NumberLabel";
+import SortIcon from "../components/SortIcon";
+import GroupsByTermTableRowRecursive from "../components/GroupsByTermTableRowRecursive";
+import { NamesMixin } from "../lib/mixins/names";
+import { TableHeaderMixin } from "../lib/mixins/table-header";
+
+export default {
+  name: "GroupsByTerm",
+  components: {
+    NumberLabel,
+    SortIcon,
+    GroupsByTermTableRowRecursive
+  },
+  mixins: [NamesMixin, TableHeaderMixin],
+  props: {
+    json: {
+      type: Array,
+      default: () => []
+    },
+    groups: {
+      type: Array,
+      default: () => []
+    },
+    options: {
+      type: Object,
+      default: () => {}
+    }
+  },
+  data() {
+    return {
+      labelProgress: I18n.t("gobierto_plans.plan_types.show.progress") || "",
+      columns: [],
+      lastLevel: 0
+    };
+  },
+  computed: {
+    termsSorted() {
+      const id = this.currentSortColumn;
+      const sort = this.currentSort;
+      return this.groups
+        .slice()
+        .sort(({ [id]: termA }, { [id]: termB }) =>
+          sort === "up"
+            ? typeof termA === "string"
+              ? termA.localeCompare(termB, undefined, { numeric: true })
+              : termA > termB ? -1 : 1
+            : typeof termA === "string"
```

Comment:
This piece of code deserves a comment

Reply:
Here you got https://github.com/PopulateTools/gobierto/pull/3199/commits/d2ce3a27e8eca8908c90c943a32e9fcd044b9719